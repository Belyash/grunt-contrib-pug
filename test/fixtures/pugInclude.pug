include inc/head.pug

- var a = 'hello pug test'
p= a
