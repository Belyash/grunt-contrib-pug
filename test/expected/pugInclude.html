<html><head><title>TEST</title></head><body></body></html><p>hello pug test</p>